src/components/Article.jsx

```diff
@@ -1 +1,26 @@
-// export const Article = () => ();
+import React from 'react';
+import 'bulma/css/bulma.css';
+
+export const Article = ({ title, date, text }) => (
+
+  <div className="box">
+    <h2 className="title" data-cy="title">
+      {title}
+    </h2>
+
+    <p className="subtitle">
+      <time dateTime={date.toISOString().slice(0, 10)} data-cy="date">
+        {
+            date.toLocaleDateString('en-US', {
+              year: 'numeric',
+              month: 'long',
+              day: 'numeric',
+            })
```

Comment:
Creating a constant to store this value instead of overloading the JSX structure is better.

Reply:
Same for `dateTime` value